samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -42,34 +44,54 @@ private enum SupportedFileType
         /// .pdf
         /// </summary>
         Pdf,
+
+        /// <summary>
+        /// .jpg
+        /// </summary>
+        Jpg,
+
+        /// <summary>
+        /// .png
+        /// </summary>
+        Png,
+
+        /// <summary>
+        /// .tif or .tiff
+        /// </summary>
+        Tiff
     };
 
     private readonly ILogger<DocumentImportController> _logger;
     private readonly DocumentMemoryOptions _options;
+    private readonly TesseractOptions _tesseractOptions;
     private readonly ChatSessionRepository _sessionRepository;
     private readonly ChatMemorySourceRepository _sourceRepository;
     private readonly ChatMessageRepository _messageRepository;
     private readonly ChatParticipantRepository _participantRepository;
     private const string GlobalDocumentUploadedClientCall = "GlobalDocumentUploaded";
     private const string ChatDocumentUploadedClientCall = "ChatDocumentUploaded";
+    private readonly string _tesseractDataPath;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="DocumentImportController"/> class.
     /// </summary>
     public DocumentImportController(
         ILogger<DocumentImportController> logger,
         IOptions<DocumentMemoryOptions> documentMemoryOptions,
+        IOptions<TesseractOptions> tesseractOptions,
         ChatSessionRepository sessionRepository,
         ChatMemorySourceRepository sourceRepository,
         ChatMessageRepository messageRepository,
         ChatParticipantRepository participantRepository)
     {
         this._logger = logger;
         this._options = documentMemoryOptions.Value;
+        this._tesseractOptions = tesseractOptions.Value;
         this._sessionRepository = sessionRepository;
         this._sourceRepository = sourceRepository;
         this._messageRepository = messageRepository;
         this._participantRepository = participantRepository;
+        this._tesseractDataPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!, "tessdata");
```

Comment:
could we leave this path up to the user? similar to how we do for `ChatStore.Filesystem.FilePath` in `appsettings.json`

Reply:
also, I'm wondering if we should place this in the existing `data` folder?

Reply:
Yep, that's a good suggestion. I had originally used the naming convention within the Tesseract repo.